### ISSUE TITLE
Should be able to run on background

### ISSUE THREAD

**Issue by timeff** [finnomena/CryptoSpace#4]

**Comment by pitchmak** [comment]
@timeff is it duplicate of #2? I need more clarification.

**Comment by timeff** [comment]
@pitchmak Nope, the program icon is on the dock when the program is running. I think it ought to stay only on menu bar widget.

**Comment by pitchmak** [comment]
@timeff I see, I've fixed it on 0674fa7cb1144236a76010bd658bb3b79000ea8d

**Comment by detohm** [comment]
@timeff hold on... How can we close it then?

**Comment by pitchmak** [comment]
hahaha, it's time to add a menu
